Help/MAC_Hints.rst

```diff
@@ -1,25 +1,74 @@
 Coverage Installation Hints for macOS Users:
 ============================================
 
+pyenv
+-----
+
+Here we use pyenv, which is simply a python installer that allows
+you to install multiple python versions and switch through them.
+In case your macOS has an older version of python that coala doesn't
+support we recommend using pyenv to install and shift to a newer
+supported version. After installation, you may also want to set up
+a virtual environment using :code:`venv` or :code:`virtualenv`.
```

Comment:
`pyenv` is not deprecated, its a third party tool to manage different versions of python.

I think given that `venv` just works, there is no point creating confusion and complexity by asking to install `pyenv`. Plus, the whole point of `pyenv` is to manage different versions of python and not shift to a newer supported version.